### PR TITLE
updated docs of install knative-gcp and examples of cloudstoragesource

### LINKED ISSUE #975: Add script which automate the set up for CloudStorageSource and update related docs
**Problem**
For CloudStorageSource, user needs to enable the Cloud Storage API and give Google Cloud Storage permissions to publish to GCP Pub/Sub. The set up procedure is kind of cumbersome. We can write a script which automate the set up for CloudStorageSource. 

**Exit Criteria**
A script which automate the set up for CloudStorageSource including enabling the Cloud Storage API and give Google Cloud Storage permissions to publish to GCP Pub/Sub(similar like https://github.com/google/knative-gcp/blob/master/hack/init_control_plane.sh and https://github.com/google/knative-gcp/blob/master/hack/init_data_plane.sh). We also need to update the related docs.

**Additional context (optional)**
Add any other context about the feature request here.

### PR DESCRIPTION
**Pull request by capri-xiyue**
Fixes #975 

## Proposed Changes

- updated docs of install knative-gcp 
-  examples of cloudstoragesource
-

### CODE REVIEW

File: docs/install/install-knative-gcp.md [hunk 1]
```diff
@@ -97,10 +90,42 @@ wish to configure the auth manually, refer to
 
   **_Note_**: If you didn't enable Workload Identity when you created your
   cluster, this step may take a long time to finish.
+  **_Note_**: This script will need four parameters
```
Comment (yolocs) [inline docs/install/install-knative-gcp.md:93]: Optional parameters available.

File: docs/install/install-knative-gcp.md [hunk 2]
```diff
@@ -97,10 +90,42 @@ wish to configure the auth manually, refer to
...
+  ```
 
 * Option 2: Export service account keys and store them as Kubernetes Secrets.
   Apply [init_control_plane.sh](../../hack/init_control_plane.sh):
 
   ```shell
   ./hack/init_control_plane.sh
   ```
+
+  **_Note_**: This script will need one parameter
```
Comment (yolocs) [inline docs/install/install-knative-gcp.md:121]: Optional parameters available.

File: docs/examples/cloudstoragesource/README.md [hunk 3]
```diff
@@ -12,12 +12,51 @@ Object Notifications for when a new object is added to Google Cloud Storage
...
-
-   ```shell
-   gcloud services enable storage-component.googleapis.com
-   gcloud services enable storage-api.googleapis.com
-   ```
+1. Enable the `Cloud Storage API` on your project and give Google Cloud Storage
+   permissions to publish to GCP Pub/Sub. Currently, we support two methods:
+   automated script or manual
+   
+   - Option 1(via automated script):
```
Comment (yolocs) [inline docs/examples/cloudstoragesource/README.md:19]: Mark one option as recommended. It helps guide the readers.

File: docs/install/install-knative-gcp.md [hunk 4]
```diff
@@ -72,23 +72,16 @@ information about Workload Identity, please see
 1. Your gcloud `CLI` are up to date. You may use `gcloud components update` to
    update it.
 
-**_Note_**: Both scripts will have a step to create a Google Cloud Service
-Account `cloud-run-events`. Ignore the error message if you already had this
-service account (error for 'service account already exists').
-TODO([#896](https://github.com/google/knative-gcp/issues/896)) Get rid of the
```
Comment (liu-cong) [inline docs/install/install-knative-gcp.md:78]: Let's also delete the TODO here: https://github.com/google/knative-gcp/blob/master/docs/install/install-knative-gcp.md

Comment (capri-xiyue) [reply docs/install/install-knative-gcp.md:78]: The one I deleted is the same as the one you linked, right?

File: docs/examples/cloudstoragesource/README.md (hunk 3 above)
Comment (yolocs) [reply docs/examples/cloudstoragesource/README.md:19]: I didn't see which option is recommended...

File: docs/install/install-knative-gcp.md (hunk 4 above)
Comment (liu-cong) [reply docs/install/install-knative-gcp.md:78]: Sorry my bad, for some reason I thought I added the TODO in 2 places. Thankfully I didn't :)

File: docs/examples/cloudstoragesource/README.md (hunk 3 above)
Comment (liu-cong) [reply docs/examples/cloudstoragesource/README.md:19]: This is not specific to this PR, but I think in general our installation docs include too much information with many levels of options. They can be intimidating to new users. Maybe we should document the recommended path in the main doc, and link details separately for users who want to tweak their installation.

But to do that we first need to have a good opinionated  recommendation :) We can follow up later.

**Comment by liu-cong** [comment]
/lgtm

Free feel to unhold

**Pull request merged**